### ISSUE TITLE
No context mapper options except export?

### ISSUE THREAD

**Issue by worthy7** [ContextMapper/vscode-extension#33]
When opening up the command menu, there is only one option

![image](https://github.com/ContextMapper/vscode-extension/assets/4762399/54e4a115-89de-4ca8-935a-fbebe1190944)

Where are the other options? Like refactor etc.

**Comment by worthy7** [comment]
Specifically where are the ‘OOAD transformation: Derive Subdomain from user requirements` ?

It says that these features already exist....
https://contextmapper.org/docs/ide/

**Comment by stefan-ka** [comment]
Hi @worthy7 

Refactorings and OOAD transformations are context-specific actions than are suggested with light-bulbs, where applicable:

![Screenshot from 2023-08-23 12-59-50](https://github.com/ContextMapper/vscode-extension/assets/6763278/d3d1b9d6-20e8-42a3-93e0-ac50311cc3f8)

By clicking on the light-bulb, you can apply the action:

![Screenshot from 2023-08-23 13-00-04](https://github.com/ContextMapper/vscode-extension/assets/6763278/8e4d8ad0-078a-40cf-959c-7d42334cb8e7)

So, long story short: the answer depends on the context.

But I definitely agree that we can approve the documentation here.

Does that already help?

Best regards,
Stefan

**Comment by worthy7** [comment]
Thanks will give it a shot
